en-US/mozorg/internet-health.lang

```diff
@@ -64,6 +64,7 @@ Help keep the Web in the hands of many
 When just a few organizations and governments control the majority of online content and access, the vital flow of ideas and knowledge is blocked. We champion an internet that belongs to the people.
 
 
+# String is obsolete. Do NOT remove.
```

Comment:
The string has been removed, it's not obsolete. We mark it as obsolete when it's replaced by a string behind a tag. You should remove both strings from this file.